src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -453,6 +453,7 @@ private async void OnSyncTimerTick(object state)
 
         private void InitializeHttp()
         {
+            _logger.LogDebug("Initializing HttpRequestManager");
```

Comment:
Does this break some tests on relying on start up logs?

Reply:
Assuming existing tests are ok as CI build is green.